src/stim/py/stim.pybind.cc

```diff
@@ -415,9 +395,85 @@ PYBIND11_MODULE(STIM_PYBIND11_MODULE_NAME, m) {
 
     m.def("_UNSTABLE_raw_gate_data", &raw_gate_data);
     m.def("_UNSTABLE_raw_format_data", &raw_format_data);
-    pybind_circuit_after_types_all_defined(c_circuit);
-    pybind_tableau_iter_after_types_all_defined(m, c_tableau_iter);
-    pybind_dem_sampler_after_types_all_defined(m, c_dem_sampler);
-    pybind_detector_error_model_after_types_all_defined(m, c_detector_error_model);
+}
+
+
+PYBIND11_MODULE(STIM_PYBIND11_MODULE_NAME, m) {
+    m.attr("__version__") = xstr(VERSION_INFO);
+    m.doc() = R"pbdoc(
+        Stim: A fast stabilizer circuit library.
+    )pbdoc";
+
+    // CAUTION: The ordering of these is important!
```

Comment:
This comment is now obsolete, right? Or at least it could be reduced to "don't mix classes and methods"

Reply:
yeah. Reduced the severity. Kept the explanation of why classes are registered before functions / methods